runtime/oti/j9nonbuilder.h

```diff
@@ -3016,6 +3018,8 @@ typedef struct J9ArrayClass {
 	/* Added temporarily for consistency */
 	UDATA flattenedElementSize;
 #endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
+	UDATA gcPriorityScanField1;
```

Comment:
rename these fields. 'priority' is too generic. 'selfReferencing' is more specific